net2/main.js

```diff
@@ -27,34 +27,19 @@ let bone = require("../lib/Bone.js");
 
 let firewalla = require("./Firewalla.js");
 
+let ModeManager = require('./ModeManager.js');
+
 // api/main/monitor all depends on sysManager configuration
 var SysManager = require('./SysManager.js');
 var sysManager = new SysManager('info');
 var fs = require('fs');
 var config = JSON.parse(fs.readFileSync('./config.json', 'utf8'));
 
-let fConfig = require('./config.js').getConfig();
 
 if(!bone.isAppConnected()) {
   log.info("Waiting for cloud token created by kickstart job...");
 }
 
-
-/*
- * Create the secondary interface 
- */
-
-var secondaryInterface = require("./SecondaryInterface.js");
-secondaryInterface.create(config,(err,ip,subnet,ipnet,mask)=>{
-    if (err == null) {
-        log.info("Successful Created Secondary Interface");
-        sysManager.secondaryIp = ip;
-        sysManager.secondarySubnet = subnet; 
-        sysManager.secondaryIpnet = ipnet; 
-        sysManager.secondaryMask  = mask; 
-    }
-});
-
```

Comment:
make sure secondary interface is always created.  Reason for this is, in case our customer have issues, we can always let them try this interface. (static ip)  This is useful for some hosts that just don't like to be compatible.